core/src/main/java/org/frankframework/pipes/AsyncSenderWithListenerPipe.java

```diff
@@ -236,47 +236,47 @@ public void setCorrelationIDXPath(String string) {
 		correlationIDXPath = string;
 	}
 
-	/** Namespace defintions for correlationIDXPath. Must be in the form of a comma or space separated list of <code>prefix=namespaceUri</code>-definitions */
+	/** Namespace definitions for correlationIDXPath. Must be in the form of a comma or space separated list of <code>prefix=namespaceUri</code> definitions. */
 	public void setCorrelationIDNamespaceDefs(String correlationIDNamespaceDefs) {
 		this.correlationIDNamespaceDefs = correlationIDNamespaceDefs;
 	}
 
-	/** Key of a PipelineSession-variable. If specified, the value of the PipelineSession variable is used as input for the XPathExpression or stylesheet, instead of the current input message */
+	/** Key of a PipelineSession variable. If specified, the value of the PipelineSession variable is used as input for the XPathExpression or stylesheet, instead of the current input message. */
 	public void setCorrelationIDSessionKey(String string) {
 		correlationIDSessionKey = string;
 	}
 
-	/** Stylesheet to extract label from message */
+	/** Stylesheet to extract a label from a message. */
 	public void setLabelStyleSheet(String string) {
 		labelStyleSheet = string;
 	}
 
-	/** XPath expression to extract label from message */
+	/** XPath expression to extract the label from the message. */
 	public void setLabelXPath(String string) {
 		labelXPath = string;
 	}
 
-	/** Namespace definitions for labelXPath. Must be in the form of a comma or space separated list of <code>prefix=namespaceUri</code>-definitions */
+	/** Namespace definitions for labelXPath. Must be in the form of a comma or space-separated list of <code>prefix=namespaceUri</code> definitions. */
 	public void setLabelNamespaceDefs(String labelXNamespaceDefs) {
 		this.labelNamespaceDefs = labelXNamespaceDefs;
 	}
 
-	/** XPath expression to extract audit trail from message */
+	/** XPath expression to extract the audit trail from the message. */
 	public void setAuditTrailXPath(String string) {
 		auditTrailXPath = string;
 	}
 
-	/** Namespace defintions for auditTrailXPath. Must be in the form of a comma or space separated list of <code>prefix=namespaceUri</code>-definitions */
+	/** Namespace definitions for auditTrailXPath. Must be in the form of a comma or space-separated list of <code>prefix=namespaceUri</code> definitions. */
 	public void setAuditTrailNamespaceDefs(String auditTrailNamespaceDefs) {
 		this.auditTrailNamespaceDefs = auditTrailNamespaceDefs;
 	}
 
-	/** Key of a PipelineSession-variable. If specified, the value of the PipelineSession variable is used as audit trail (instead of the default 'no audit trail') */
+	/** Key of a PipelineSession variable. If specified, the value of the PipelineSession variable is used as an audit trail (instead of the default 'no audit trail'). */
```

Comment:
```suggestion
	/** Key of a pipeline session variable. If specified, the value of the PipelineSession variable is used as an audit trail (instead of the default 'no audit trail'). */
```